proto/api/v2/compliance_scan_configuration_service.proto

```diff
@@ -86,6 +87,46 @@ message ComplianceRunReportResponse {
   string error_msg = 3;
 }
 
+message ComplianceReportHistoryRequest {
+  string id = 1;
+  RawQuery report_param_query = 2;
+}
+
+message ComplianceReportHistoryResponse {
+  repeated ComplianceReportSnapshot compliance_report_snapshots = 1;
+}
+
+message ComplianceReportSnapshot {
+  string snapshot_id = 1;
```

Comment:
In the VM Reports context we use `report_job_id` and `report_config_id`. Here we use `snapshot_id` and `scan_config_id`. I think `scan_config_id` is fine. I'm wondering about that `snapshot_id` though. Would calling it `report_job_id` make sense here? Either that or renaming `report_job_id` to `snapshot_id` in the `report_service.proto`. Let me know your thoughts

Reply:
My thought process was: `report_job_id` makes sense in VM because we (central) are the ones triggering and handling the report jobs. In compliance we don't have jobs to be triggered by central, we passively listen to the compliance scans and and once they're done we capture the snapshot and report it, thus the name `snapshot_id`. 

AFAIK VM is in GA so we shouldn't change the protos unless we deprecate them first. If you think it's better to have parity in the naming, I'd change the compliance proto from `snapshot_id` to `report_job_id`.

Reply:
I think having parity in naming would be good. I'll leave it to your best judgment though

Reply:
Changed to `report_job_id`